tests/filecheck/dialects/riscv/riscv_ops_custom_syntax.mlir

```diff
@@ -0,0 +1,270 @@
+  %sltu = "riscv.sltu"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  %and = "riscv.and"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  %or = "riscv.or"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  %xor = "riscv.xor"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  %sll = "riscv.sll"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  %srl = "riscv.srl"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  %sub = "riscv.sub"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  %sra = "riscv.sra"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  "riscv.nop"() : () -> ()
+  // CHECK-NEXT: %add = riscv.add %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %slt = riscv.slt %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %sltu = riscv.sltu %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %and = riscv.and %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %or = riscv.or %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %xor = riscv.xor %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %sll = riscv.sll %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %srl = riscv.srl %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %sub = riscv.sub %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %sra = riscv.sra %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: riscv.nop :
+
+  // RV32I/RV64I: 2.5 Control Transfer Instructions
+  // terminators continue at the end of module
+
+  // Unconditional Branch Instructions
+  "riscv.jal"() {"immediate" = 1 : i32} : () -> ()
+  "riscv.jal"() {"immediate" = 1 : i32, "rd" = !riscv.reg<>} : () -> ()
+  "riscv.jal"() {"immediate" = #riscv.label<"label">} : () -> ()
+  // CHECK-NEXT: riscv.jal 1 :
+  // CHECK-NEXT: riscv.jal 1 None :
+  // CHECK-NEXT: riscv.jal "label" :
+
+  "riscv.j"() {"immediate" = 1 : i32} : () -> ()
+  "riscv.j"() {"immediate" = #riscv.label<"label">} : () -> ()
+  // CHECK-NEXT: riscv.j 1 :
+  // CHECK-NEXT: riscv.j "label" :
+
+  "riscv.jalr"(%0) {"immediate" = 1 : i32}: (!riscv.reg<>) -> ()
+  "riscv.jalr"(%0) {"immediate" = 1 : i32, "rd" = !riscv.reg<>} : (!riscv.reg<>) -> ()
+  "riscv.jalr"(%0) {"immediate" = #riscv.label<"label">} : (!riscv.reg<>) -> ()
+  // CHECK-NEXT: riscv.jalr %0 1 :
+  // CHECK-NEXT: riscv.jalr %0 1 None :
+  // CHECK-NEXT: riscv.jalr %0 "label" :
+
+  // Conditional Branch Instructions
+  "riscv.beq"(%0, %1) {"offset" = 1 : i32}: (!riscv.reg<>, !riscv.reg<>) -> ()
+  "riscv.bne"(%0, %1) {"offset" = 1 : i32}: (!riscv.reg<>, !riscv.reg<>) -> ()
+  "riscv.blt"(%0, %1) {"offset" = 1 : i32}: (!riscv.reg<>, !riscv.reg<>) -> ()
+  "riscv.bge"(%0, %1) {"offset" = 1 : i32}: (!riscv.reg<>, !riscv.reg<>) -> ()
+  "riscv.bltu"(%0, %1) {"offset" = 1 : i32}: (!riscv.reg<>, !riscv.reg<>) -> ()
+  "riscv.bgeu"(%0, %1) {"offset" = 1 : i32}: (!riscv.reg<>, !riscv.reg<>) -> ()
+  // CHECK-NEXT: riscv.beq %0, %1 1 :
+  // CHECK-NEXT: riscv.bne %0, %1 1 :
+  // CHECK-NEXT: riscv.blt %0, %1 1 :
+  // CHECK-NEXT: riscv.bge %0, %1 1 :
+  // CHECK-NEXT: riscv.bltu %0, %1 1 :
+  // CHECK-NEXT: riscv.bgeu %0, %1 1 :
```

Comment:
Where did the types go here?

Reply:
They have no result types.

Reply:
Then we should probably not print a colon

Reply:
The colon is used for indicating the end of attributes. Otherwise, optional attributes may parse the upcoming strings as attributes.

Reply:
Wait, we still have argument types though, right? We need to print them still.

Reply:
I'm not sure if I completely understand what do you mean. Would you mind giving a concrete example?

Reply:
Add types.